src/Surfnet/StepupMiddleware/CommandHandlingBundle/Command/RaExecutable.php

```diff
@@ -23,4 +23,9 @@
  */
 interface RaExecutable
 {
+    /**
+     * Returns ra institution for authorization purposes
+     * @return string|null
+     */
+    public function getRaInstitution();
```

Comment:
+1 for adding this, please update the phpdoc comment of the interface itself. It mentions this is a marker interface, but it not longer is. Same goes for the SelfServiceExecutable.